### PR TITLE
Add industrial ci pipeline

### PR DESCRIPTION
**Pull request by fmauch**
as per subject

### CODE REVIEW

**Comment by fmauch** [comment]
@stefanscherzinger Would you like to test other distributions, as well?

**Comment by stefanscherzinger** [comment]
I would like to test Kinetic, even when its end of life is soon..

File: .github/workflows/industrial_ci_action.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    strategy:
+      matrix:
+        env:
+          - {ROS_DISTRO: melodic, ROS_REPO: testing}
+          - {ROS_DISTRO: melodic, ROS_REPO: main}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: 'ros-industrial/industrial_ci@master'
+        env: ${{matrix.env}}
```
Comment (gavanderhoorn) [inline .github/workflows/industrial_ci_action.yml]: I believe that syntax isn't quite idiomatic for `matrix` in GHA: https://github.com/ros-industrial/industrial_ci/issues/590.

That issue also documents the alternative structure which is.

Comment (fmauch) [reply .github/workflows/industrial_ci_action.yml]: Thanks, that is actually nicer!

**Review by stefanscherzinger — APPROVED** [review]

**Pull request merged**